recipes/gluonts/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "gluonts" %}
+{% set version = "0.11.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gluonts-{{ version }}.tar.gz
+  sha256: c5db67304853db450620edf2a5bf248eb7e6ee65e4290641bf725e4a2e7d00db
+
+build:
+  skip: true  # [py2k]
```

Comment:
You don't need to skip py2k in conda-forge. You may consider trying to make this noarch though by dropping dataclasses and py36.

Reply:
@ocefpaf Thanks for the suggestion. I have dropped dataclasses for py36 and changed to noarch.